azure-pipelines.yml

```diff
@@ -182,22 +182,6 @@ stages:
               AZDO_BASE_URL: $(AzdoBaseUrl)
               ITS_ORGA: $(AzdoItsOrga)
               ITS_PROJECT_NAME: $(AzdoItsProjectName)
-              ITS_DOTNET_PIPELINE_NAME: $(AzdoItsDotnetPipelineName)
-              ITS_DOTNET_SC_PROJECT_KEY: $(SonarCloudItsDotnetProjectKey)
-              ITS_DOTNET_EXPECTED_COVERAGE: $(DotnetExpectedCoverage)
-              ITS_DOTNET_EXPECTED_NCLOCS: $(DotnetExpectedNcLocs)
-              ITS_MAVEN_PIPELINE_NAME: $(AzdoItsMavenPipelineName)
-              ITS_MAVEN_SC_PROJECT_KEY: $(SonarCloudItsMavenProjectKey)
-              ITS_MAVEN_EXPECTED_COVERAGE: $(MavenExpectedCoverage)
-              ITS_MAVEN_EXPECTED_NCLOCS: $(MavenExpectedNcLocs)
-              ITS_GRADLE_PIPELINE_NAME: $(AzdoItsGradlePipelineName)
-              ITS_GRADLE_SC_PROJECT_KEY: $(SonarCloudItsGradleProjectKey)
-              ITS_GRADLE_EXPECTED_COVERAGE: $(GradleExpectedCoverage)
-              ITS_GRADLE_EXPECTED_NCLOCS: $(GradleExpectedNcLocs)
-              ITS_SCANNERCLI_PIPELINE_NAME: $(AzdoItsScannerCliPipelineName)
-              ITS_SCANNERCLI_SC_PROJECT_KEY: $(SonarCloudItsCLIProjectKey)
-              ITS_SCANNERCLI_EXPECTED_COVERAGE: $(CobolExpectedCoverage)
-              ITS_SCANNERCLI_EXPECTED_NCLOCS: $(CobolExpectedNcLocs)
```

Comment:
I am missing the rational to not use the pipeline env vars, Is it just that the values are now hardcoded in the TestCases directly?

Reply:
Yes values are now hardcoded in test cases

Before this PR, all values were:
- Set in AzDO in a variable group, each one manually
- Forwarded to the project with env variables, each one manually (this piece of code)
- Read by the C# codebase

But:
- Each one of these variables is used in a single location, so there's no use in using variables
- We now just have to create a single file in `its/Models/` and include it in the main file to create a new E2E pipeline

Reply:
if it's nothing secret, sounds fine to remove them. If they were somehow re-used it could also make sense but since it's not; your rational makes sense 👍🏻